render/go/renderer.go

```diff
@@ -205,14 +209,17 @@ func MustLoadGinServerTestConfig(configPath, openAPIPath string) config.Data {
 		OpenAPIFile: openAPIPath,
 		OutputPath:  ".",
 	}
-	if err := c.Load(); err != nil {
+	if err := c.Load(configFileReader); err != nil {
 		panic(err)
 	}
 	return c
 }
 func MustLoadGoClientTestConfig(configPath, openAPIPath string) config.Data {
 	// TODO: `Must` this function and load the returned config before returning it
 	config.ConfigFilePath = configPath
+	absCfgPath, _ := filepath.Abs(config.ConfigFilePath)
```

Comment:
Errors?

Reply:
I was relying on the panic below. These errors would have meant Load can't work. But I've handled them explicitly. Should be more clear now.